src/view/MainViewManager.js

```diff
@@ -845,6 +846,35 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Switch between panes
+     */
+    function switchPaneFocus() {
+        var $firstPane = $('#first-pane'), $secondPane = $('#second-pane');
+        if($firstPane.hasClass('active-pane')) {
+            $secondPane.click();
+        }
+        else {
+            $firstPane.click();
+        }
+    }
+
+    /**
+     * Unit test for switching from first pane to second pane
```

Comment:
Hi @petetnt  I'm having trouble simulating 'Alt-w' key press using SpecRunnerUtils.simulateKeyEvent. Here is my attempt:

`SpecRunnerUtils.simulateKeyEvent(KeyEvent.DOM_VK_ALT, "keydown", $('#first-pane')[0]);
                    SpecRunnerUtils.simulateKeyEvent(KeyEvent.DOM_VK_W, "keydown", $('#first-pane')[0]);
                    SpecRunnerUtils.simulateKeyEvent(KeyEvent.DOM_VK_W, "keypress", $('#first-pane')[0]);
                    SpecRunnerUtils.simulateKeyEvent(KeyEvent.DOM_VK_W, "keyup", $('#first-pane')[0]);`

Could you please let me know what I'm doing wrong? Thank you so much!